docs/examples/button/group.vue

```diff
@@ -11,6 +11,12 @@
     <el-button type="primary" :icon="Share" />
     <el-button type="primary" :icon="Delete" />
   </el-button-group>
+
+  <el-button-group class="ml-4" :vertical="true">
```

Comment:
It seems that direction is mostly used in element components to display arrangement directions.
![image](https://github.com/user-attachments/assets/89f07061-280a-4594-9e5c-01896945d2e2)